openapi/src/common/schemas.yaml

```diff
@@ -0,0 +1,232 @@
+# Common schemas
```

Comment:
We can connect the comments to the nodes they care about by using a node that's ignored by swagger like `x-comment`. i'm looking for the exact documentation of this...

Reply:
It would also have to be ignored by the validator

Reply:
I think it was when I tried it.

Reply:
OK, that reference is from V2. They are now called [Specification Extensions](https://swagger.io/specification/)
I'm going to attach that to the PF-1378 ticket